res/scripts/uninstall.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Function to display usage instructions
+usage() {
+  echo "Usage: $0 -s <service_name> [-c]"
+  echo "  -s: Name of the Docker service to remove"
+  echo "  -c: Optional flag to clean up unused containers, images, volumes, and networks"
+  exit 1
+}
+
+CLEAN_UP=false
+
+# Parse command line arguments
+while getopts "s:c" opt; do
+  case $opt in
+    s) SERVICE_NAME=$OPTARG ;;
+    c) CLEAN_UP=true ;;
+    *) usage ;;
+  esac
+done
+
+if [ -z "$SERVICE_NAME" ]; then
+  usage
+fi
+
+# Check if Docker is running
+if ! systemctl is-active --quiet docker; then
+  echo "Docker is not running. Please start Docker before running this script."
+  exit 1
+fi
+
+echo "Stopping and removing the Docker service: $SERVICE_NAME"
+docker service rm "$SERVICE_NAME"
+
+if [ $? -eq 0 ]; then
```

Comment:
```suggestion

if docker service rm "$SERVICE_NAME"; then
```
to follow [SC2181](https://github.com/koalaman/shellcheck/wiki/SC2181)